pysatModels/models/pydineof_dineof.py

```diff
@@ -208,12 +206,19 @@ def download(date_array=None, tag=None, inst_id=None, data_path=None, **kwargs):
                                                            day=date.day))
         remote_path = '/'.join((remote_url.strip('/'), remote_path.strip('/'),
                                 fname))
-        req = requests.get(remote_path)
-        if req.status_code != 404:
-            open(saved_local_fname, 'wb').write(req.content)
-        else:
-            warnings.warn('Unable to find remote file: {:}'.format(remote_path))
+        with requests.get(remote_path) as req:
+            if req.status_code != 404:
+                if not os.path.isfile(saved_local_fname):
+                    with open(saved_local_fname, 'wb') as open_f:
+                        open_f.write(req.content)
+            else:
+                warnings.warn('Unable to find remote file: {:}'.format(
+                    remote_path))
     else:
+        saved_local_fname = os.path.join(data_path, 'empty_file.txt')
+        with open(saved_local_fname, 'w') as open_f:
+            open_f.write(' ')
```

Comment:
Why do we create a file if there's nothing to download?

Reply:
Good question. 🐵 

It has been awhile but they may be some leftover test code intended to run on my local system... If so, then I must've forgot about it, and didn't see it when I copied code over from the other branch. 🙈